envs/quadrotors/quad_multi_model.py

```diff
@@ -0,0 +1,60 @@
+import torch
+from torch import nn
+
+from algorithms.appo.model_utils import get_obs_shape, nonlinearity, create_standard_encoder, EncoderBase, \
+    register_custom_encoder
+from algorithms.utils.pytorch_utils import calc_num_elements
+from utils.utils import log
+
+
+class QuadMultiMeanEncoder(EncoderBase):
+    '''Mean embedding encoder based on the DeepRL for Swarms Paper'''
```

Comment:
Please adhere to the code style in the rest of the codebase. We use single quotes ' for all strings, and triple double quotes for docstrings.
I have no particular preference for either, just for consistency.